sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -281,4 +283,41 @@ public partial class KeyVaultSelectiveKeyRestoreResult
         public System.DateTimeOffset EndTime { get { throw null; } }
         public System.DateTimeOffset StartTime { get { throw null; } }
     }
+    public partial class KeyVaultSetting
+    {
+        internal KeyVaultSetting() { }
+        public string Name { get { throw null; } }
+        public Azure.Security.KeyVault.Administration.SettingType? Type { get { throw null; } }
+        public string Value { get { throw null; } }
+    }
+    public partial class KeyVaultSettingsClient
+    {
+        protected KeyVaultSettingsClient() { }
+        public KeyVaultSettingsClient(System.Uri vaultUri, Azure.Core.TokenCredential credential) { }
+        public KeyVaultSettingsClient(System.Uri vaultUri, Azure.Core.TokenCredential credential, Azure.Security.KeyVault.Administration.KeyVaultAdministrationClientOptions options) { }
+        public virtual System.Uri VaultUri { get { throw null; } }
+        public virtual Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting> GetSetting(string name, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting>> GetSettingAsync(string name, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting[]> GetSettings(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting[]>> GetSettingsAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting> UpdateSetting(string name, bool value, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting>> UpdateSettingAsync(string name, bool value, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SettingType : System.IEquatable<Azure.Security.KeyVault.Administration.SettingType>
```

Comment:
Should this be named `KeyVaultSettingType` if the public type is `KeyVaultSetting`?

Reply:
We have `KeyType` to `KeyVaultKey` so I think `SettingType` makes more sense...if we even keep it.

Reply:
We actually have `KeyVaultRoleType` in Java, I can't recall the exact reasoning behind the name but I think it's because all other classes it relates to start with `KeyVaultRole`. Maybe it was because RoleType is something that can be found elsewhere for any library that offers RBAC functionality. In any case, I think for Java we will stick to precedent and .NET can probably do the same. I'll talk to Jonathan or Srikanta about this offline.